src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32.cs

```diff
@@ -32,10 +32,16 @@ public class Win32 : IProgram
         public bool Enabled { get; set; }
         public bool hasArguments { get; set; } = false;
         public string Location => ParentDirectory;
+        public string AppType { get; set; }
 
         private const string ShortcutExtension = "lnk";
         private const string ApplicationReferenceExtension = "appref-ms";
         private const string ExeExtension = "exe";
+        private const string InternetShortcutExtension = "url";
+
+        private const string InternetShortcutApplication = "Internet Shortcut application";
```

Comment:
We should remove "application" here. It's only an web link and not an application like PWAs or pinned website.